### DISCUSSION TITLE
从Follower恢复元数据后数据表Tablet受损

### DISCUSSION THREAD

**Discussion by leiyangcd** [apache/doris#12537] [A - General / Q&A]
之前因异常断电导致follower元数据受损，从其他节点恢复后，总是出现数据表的Tablet受损，目前两次元数据恢复都出现了这样的问题，不知原因为何，报错信息：
1064 - errCode = 2, detailMessage = failed to initialize storage reader. tablet=76274.828796829.e74e3b763a26caae-148d4fe1bdfb56bc, res=-230, backend=192.168.xx.xx

**Comment by JNSimba** [comment]
what is the doris version？

**Reply by leiyangcd** [reply]
0.15

**Reply by JNSimba** [reply]
Upgrade to 1.1.2 and try？
